packages/calypso-config/src/desktop.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable wpcalypso/import-docblock */
+import type { ConfigData } from '@automattic/create-calypso-config';
+
+// TODO: Revisit whether it is useful for the Desktop app to override the following properties:
+// signup_url, login_url, logout_url and discover_logged_out_redirect_url
+
+const config = {
```

Comment:
It feels wrong to have these values hardcoded in a package. Could you expand on why this is needed?

Reply:
One alternative is to have these values loaded from the Main process to `window.electron` via `preload.js`, which might involve some modifications to the build scripts as well. If that is a little too involved we can leave as a TODO and return to this as a separate task.